File: go/test/protocol/test_case_mqtt_config.go
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.
package protocol

type TestCaseMqttConfig struct {
	ClientID *string `yaml:"client-id"`
}

func MakeTestCaseMqttConfig() TestCaseMqttConfig {
	return TestCaseMqttConfig{
		nil,
	}
}

File: rust/azure_iot_operations_protocol/tests/metl/test_case_mqtt_config.rs
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

use serde::Deserialize;

#[derive(Clone, Deserialize, Debug)]
#[allow(dead_code)]
pub struct TestCaseMqttConfig {
    #[serde(rename = "client-id")]
    pub client_id: Option<String>,
}

impl TestCaseMqttConfig {
    pub fn get_default() -> Self {
        Self { client_id: None }
    }
}
